lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -762,7 +785,9 @@ class MultipacketSession {
   addSegment(segment) {
     debug(`*** Got a Multipacket Segment: ${segment.packetNumber + 1} of ${this.packetsToFetch}, count: ${this.segmentsFilled + 1}`);
     if (this.packetsToFetch === this.segmentsFilled) {
-      throw new InvalidStateError('Segments already filled. Should not be getting duplicates.');
+      // TODO: Should this throw, or just debug?
```

Comment:
TODO for when? Before beta?

Reply:
Yes, before beta. This is a change I made for an alpha user.